### DISCUSSION TITLE
is there a way to open terminal from inside yazi?

### DISCUSSION THREAD

**Discussion by yuvalsp-pelles** [sxyazi/yazi#1649] [1. Q&A]
.

**Comment by xfzv** [comment]
https://yazi-rs.github.io/docs/tips/#dropping-to-shell maybe?

**Comment by ze0987** [comment]
Or for example:
`{ on = "<C-t>", run ='shell "foot" --orphan --confirm' , desc = "Open foot terminal here" },`
